packages/core/services/mongodb-remote/src/internal/ResultDecoders.ts

```diff
@@ -88,15 +101,26 @@ class RemoteDeleteResultDecoder implements Decoder<RemoteDeleteResult> {
   }
 }
 
-class ChangeEventDecoder<T> implements Decoder<ChangeEvent<T>> {
+class UpdateDescriptionDecoder implements Decoder<UpdateDescription> {
+  public decode(from: any) {
```

Comment:
Good catch. TypeScript doesn't require return types since it's intended to be compatible with JS, but if you exclude them, it makes it more likely that the compiler will miss type-unsafe code.